qemu/tests/cfg/seabios_reboot_timeout.cfg

```diff
@@ -24,6 +24,5 @@
             boot_reboot_timeout = 65535
         - time65536:
             required_qemu = [, 4)
-            Host_RHEL.m8:
-                only Host_RHEL.m8.u0
+            only Host_RHEL.m6 Host_RHEL.m7 Host_RHEL.m8.u0
```

Comment:
At fist, it is puzzled this case does not be cancelled with qemu 5.2, since it already restrict the qemu version. After checked again, i got it was overwritten by the setting in guest-hw.cfg when use `rng_builtin`, this is the reason for this issue when checking the debug.log:
https://github.com/avocado-framework/avocado-vt/blob/295a165e271a2518871dee3b27985bf2f8d45f12/virttest/shared/cfg/guest-hw.cfg#L442

Suggest explain the reason with more details.